src/Apps/Artist/Routes/Overview/Components/ArtworkFilter/index.tsx

```diff
@@ -235,40 +273,70 @@ class Filter extends Component<Props> {
   }
 }
 
-export const ArtworkFilterFragmentContainer = createFragmentContainer(Filter, {
-  artist: graphql`
-    fragment ArtworkFilter_artist on Artist
-      @argumentDefinitions(
-        medium: { type: "String", defaultValue: "*" }
-        major_periods: { type: "[String]" }
-        partner_id: { type: "ID" }
-        for_sale: { type: "Boolean" }
-        aggregations: {
-          type: "[ArtworkAggregation]"
-          defaultValue: [MEDIUM, TOTAL, GALLERY, INSTITUTION, MAJOR_PERIOD]
-        }
-      ) {
-      id
-      filtered_artworks(
-        aggregations: $aggregations
-        medium: $medium
-        major_periods: $major_periods
-        partner_id: $partner_id
-        for_sale: $for_sale
-        size: 0
-      ) {
-        aggregations {
-          slice
-          counts {
-            name
-            count
-            id
+export const ArtworkFilterRefetchContainer = createRefetchContainer(
+  (props: Props) => {
+    return (
+      <Subscribe to={[FilterState]}>
```

Comment:
This 'wrapper' is what used to be a full on Query Renderer.

Now, it just listens to a new filter and re-renders the entire thing. We pass the new filter in as props so we can use `componentDidUpdate` to see if any filter changed, and thus trigger a refetch and render. That's also why I switched this from a fragment to a refetch container.